conans/client/runner.py

```diff
@@ -107,12 +107,12 @@ def get_stream_lines(the_stream):
 
     def _simple_os_call(self, command, cwd):
         if not cwd:
-            return os.system(command)
+            return subprocess.call(command, shell=isinstance(command, six.string_types))
         else:
             try:
                 old_dir = get_cwd()
                 os.chdir(cwd)
-                result = os.system(command)
+                result = subprocess.call(command, shell=isinstance(command, six.string_types))
```

Comment:
Here we should probably write all the output to `STDOUT`. Same above.

Reply:
Am I right that this would be a change in behaviour? If I'm reading the documentation of `os.system()` right, it doesn't perform redirection of any kind (neither does `system(3)`).

It's a simple enough change to make, just want to confirm if it should be done as part of this PR or a separate one.

Reply:
IMO it is not breaking, we are free to change messages and add lines to the regular output, showing output from commands should be ok (even desired from my POV).

But I'm ok if we leave it as it is now, all the Conan output needs some rework and we can decide about it at that moment.